packages/connect/src/data/connectSettings.ts

```diff
@@ -128,5 +128,9 @@ export const parseConnectSettings = (input: Partial<ConnectSettings> = {}) => {
         settings.manifest = parseManifest(input.manifest);
     }
 
+    if (typeof input.sharedLogger === 'boolean') {
+        settings.sharedLogger = input.sharedLogger;
+    }
+
```

Comment:
maybe it should get its default on line 13 in this file too?

Reply:
d9733a0c96d72609bd4414667cd4251696d88d04